tests/subsys/storage/stream/stream_flash/testcase.yaml

```diff
@@ -12,5 +12,5 @@ tests:
     tags: stream_flash
   storage.stream_flash.mpu_allow_flash_write:
     extra_args: OVERLAY_CONFIG=mpu_allow_flash_write.overlay
-    platform_allow:  nrf52840_pca10056
+    platform_allow: nrf52840dk_nrf52840
```

Comment:
I built and tested this test scenario on real hardware and it failed. 
@hakonfam could you look at this test, before merge this change into main?

Reply:
Will do

Reply:
This test passes locally on my HW (v2.0.1)

Reply:
It also works on my local setup.

Reply:
Hmmmm, I was also checking on board v2.0.1 with Zephyr SDK: `zephyr-sdk-0.13.1` and it still failed. I will check my nrfjprog version and other things, but if you wrote, that it works on yours boards I will no longer block this PR. 
@hakonfam @PerMac - thanks for checking :)